src/components/breadcrumb-bar/style.module.css

```diff
@@ -0,0 +1,44 @@
+/* Note: the ds- type style comes from the global design system.
+   Once we have a way to consume these styles from a more
+   central location, we should switch to that approach. */
+.ds-type-body-small {
+  font-size: 13px;
+  line-height: 18px;
+}
```

Comment:
Want to add this to a `typography.css` file [in `src/styles`](https://github.com/hashicorp/dev-portal/tree/assembly-ui-v1/src/styles)?

Reply:
✅ Done, thanks for flagging! I've added a `typography.css` file with a lil todo there about pulling in the `--tokens` from the `hashicorp/design-system-tokens` `npm` package, but figure we leave that outside the scope of this PR to avoid yak-shaving. Let me know what you think!